site/app/templates/officeHoursQueue/SearchStudentQueue.twig

```diff
@@ -0,0 +1,117 @@
+<div id='student_queue_history'>
+    <br>
+    <h2>Student's Queue History</h2>
+    <p>Student ID</p>
+    <div id='search-student-queue'>
+        <input id='search-student-queue-input' data-testid='search-student-queue-input' type='text' placeholder="Enter student's name">
+        <button id='search-student-queue-btn' data-testid='search-student-queue-btn' class="btn btn-primary">Search</button>
+    </div>
+    <table id='student-queue-table' class="table table-striped" style="margin-top: 15px;">
+        <caption style="padding: 5px; border: 1px solid var(--standard-light-gray); font-weight: 600;"></caption>
+        <thead>
+            <tr>
+                <th scope='col'>#</th>
+                <th scope='col'>Current state</th>
+                <th scope='col'>Queue</th>
+                <th scope='col'>Time Entered</th>
+                <th scope='col'>Time In Queue</th>
+                <th scope='col'>Time Helped</th>
+                <th scope='col'>Help Started By</th>
+                <th scope='col'>Removal Type</th>
+            </tr>
+        </thead>
+        <tbody id="search-student-tbody">
+
+        </tbody>
+    </table>
+</div>
+
+<script>
+    $('#search-student-queue-input').autocomplete({
+        source: {{ student_full|raw }}
```

Comment:
Using `|raw` is a security liability.  You can use [`| e('js')`](https://twig.symfony.com/doc/3.x/filters/escape.html) instead.

Reply:
Also— `e('js')` causes a console error here.